caluma/caluma_core/events.py

```diff
@@ -26,6 +34,9 @@ def send_event(signal, **kwargs):
     result = signal.send_robust(**kwargs)
     for func, exc in result:
         if exc is not None:
+            if func._raise_exception:
```

Comment:
This will raise the first exception, which could mask out if there were others in other event handlers. I'm ok with only raising one, but the others should at least be logged.

Maybe we could instead raise our own exception here and add the caught ones to it's properties:
```python
class SignalHandlingError(RuntimeError):
    def __init__(self, msg, exceptions):
        self.caught_exceptions = exceptions
        # super init etc omitted
```

Reply:
Nice! Should the argument then be named `raise_exceptions` (plural) again?

Reply:
IMO no, because it's max one exception per receiver.

Reply:
True, I'll leave it like this!

Reply:
Done